canonical/ubuntu.com
# Create base template with CTA footer for case studies

## Done

- Created contact form for case studies
- Added case study template paths
- Create case study base template with footer CTA
- Figma: https://www.figma.com/design/AeIJ3GsqnJCHtiBYhkTuTV/24.10-Case-study-template?node-id=0-1&t=wlZAf1AkHWJJl0TN-0

## QA

- Go to https://ubuntu-com-14126.demos.haus/case-studies
- Click on `Contact us` button on footer
- Fill up form and check payload submission
- The footer design should match Figma

## Issue / Card

Fixes [WD-13117](https://warthogs.atlassian.net/browse/WD-13117)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-13117]: https://warthogs.atlassian.net/browse/WD-13117?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
